src/api/database/models/OngoingMeetingCount.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  Column,
+  ForeignKey,
+  Table,
+  BelongsTo,
+  Model,
+  PrimaryKey
+} from "sequelize-typescript";
+import { INTEGER, STRING } from "sequelize";
+
+@Table
+class OngoingMeetingCount extends Model {
```

Comment:
Write a comment to explain this model, for those who aren't familiar with the design. Something like "A map from TencentMeeting admin user ids to the number of ongoing meetings in the user's account. Normally we expect either 0 or 1 because...."

And talking about which, we should probably `TMAdminUserId` instead as the primary key.

Reply:
`TMAdminUserId`